### PR TITLE
Accessibility

### PR DESCRIPTION
**Pull request by Joaquin123455**
### General requirements

- Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
- Make sure that you used correct [GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md).
- Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).

### HTML/CSS requirements

- Follow our list of [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

### Project requirements

_**IMPORTANT NOTE:** For each Microverse project, you need to create a separate pull request. [Read how to work with multiple pull requests in your one repository in the most efficient way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/multiple_pull_requests.md)._

- Check all of the items listed in the article [Easy Checks – A First Review of Web Accessibility](https://www.w3.org/WAI/test-evaluate/preliminary/) (which you already read in the previous lesson).
- Determine if your Portfolio needs any changes.
- Make changes in a separate branch and open a new pull request.
- Prepare a report listing all points that you have checked, and the result of your evaluation.
- Your report should include:
     -  List all points that you have checked, i.e:
        - [Page titles](https://www.w3.org/WAI/test-evaluate/preliminary/#title) no issues found
        - [Image text alternatives** (`alt text`)](https://www.w3.org/WAI/test-evaluate/preliminary/#images) issues not found 
        - [Text headings](https://www.w3.org/WAI/test-evaluate/preliminary/#headings) issues not found
        - [Color contrast](https://www.w3.org/WAI/test-evaluate/preliminary/#contrast) issue found: buttons colors changed, logo and main title color changed too.
        - [Resize](https://www.w3.org/WAI/test-evaluate/preliminary/#resize) no issues found
        - [Interaction](https://www.w3.org/WAI/test-evaluate/preliminary/#interaction)  issues not found
          (labels are optional but recommended as voiceover tools can read them, and should be set as invisible to match the Figma design)
        - [Moving content](https://www.w3.org/WAI/test-evaluate/preliminary/#moving) issues not found 
        - [Multimedia](https://www.w3.org/WAI/test-evaluate/preliminary/#media) issues not found
        - [The basic structure of the page](https://www.w3.org/WAI/test-evaluate/preliminary/#structure) issues found: change title, and change h2 to h1. 
     - Each point on the list needs to include the result of your evaluation:
        - If everything was ok, make a statement about that (e.g. "Page titles - no issues found").
        - If you needed to change something, [link to the corresponding line of code in your pull request changes](http://www.chasewoodford.com/blog/linking-highlighted-lines-code-github/#:~:text=To%20link%20to%20multiple%20lines,php%23L117%2DL148).
- Add your report to the description of your pull request.

### CODE REVIEW

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

_**I decided to make my comments here because I could not make inline comments on your code. Kindly use the screenshots to trace where changes are required.**_

- [ ] It is part of the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/6_accessability.md) to prepare a report listing all points that you have checked, and the result of your evaluation. This should be present in your PR description.
Below is the example of how to prepare your report.

```
- Page titles - no issues found. Provided not issues found, but if there's issue, you're ought to resolve that before making your report.
- Image text alternative - no issues found.
- etc. .......
```

- [ ] The title in the head section of your page is not descriptive. Something like `Joaquin's Portfolio` could be a nice option. Kindly fix that.

your-title

- [ ] This is the main title on your page, for convention, semantics reason, it should be wrapped in `h1` tag and it should only be one on a webpage. Kindly fix that.

your-main-title


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: desktop.css [hunk 1]
```diff
@@ -221,7 +224,7 @@
     font-size: 12px;
     line-height: 16px;
     letter-spacing: 0.03em;
-    color: #6070ff;
+    color: var(--my-contrast-color);
```
Comment (Joaquin123455) [inline desktop.css:227]: contrast color changed.

File: index.html [hunk 2]
```diff
@@ -4,11 +4,12 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="description" content="#">
   <link rel="stylesheet" href="style.css">
   <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
   <link rel="stylesheet" href="desktop.css">
-  <title>Document</title>
+  <title>Joaquin's Portfolio</title>
```
Comment (Joaquin123455) [inline index.html:12]: structure changed.

File: index.html [hunk 3]
```diff
@@ -26,8 +27,8 @@
   </header>
   <main>
     <section class="main-page">
-      <h2 class="main-title"> I'm Joaquin. <br>
-        Glad to see you!</h2>
+      <h1 class="main-title"> I'm Joaquin. <br>
```
Comment (Joaquin123455) [inline index.html:30]: structure changed.

File: style.css [hunk 4]
```diff
@@ -38,7 +38,7 @@ header {
   display: flex;
   align-items: center;
   text-align: center;
-  color: #6070ff;
+  color: #214073;
```
Comment (Joaquin123455) [inline style.css:41]: contrast color changed.

File: style.css [hunk 5]
```diff
@@ -58,13 +58,13 @@ header {
 .main-title {
   font-family: 'Poppins', sans-serif;
   font-style: normal;
-  font-weight: bold;
+  font-weight: 700;
```
Comment (Joaquin123455) [inline style.css:61]: contrast color changed.

File: style.css [hunk 6]
```diff
@@ -58,13 +58,13 @@ header {
...
   font-size: 40px;
   line-height: 52px;
   margin-left: 24px;
   padding-top: 104px;
   width: 80%;
-  color: #172b4d;
+  color: #214073;
```
Comment (Joaquin123455) [inline style.css:67]: contrast color changed.

File: style.css [hunk 7]
```diff
@@ -87,7 +87,7 @@ header {
   line-height: 24px;
   padding-top: 12px;
   margin-left: 24px;
-  color: #7f8cff;
+  color: #1f35ff;
```
Comment (Joaquin123455) [inline style.css:90]: contrast color changed.

File: style.css [hunk 8]
```diff
@@ -98,7 +98,7 @@ header {
...
```
Comment (Joaquin123455) [inline style.css:101]: contrast color changed.

File: style.css [hunk 9]
```diff
@@ -302,7 +302,7 @@ span {
   font-size: 12px;
   line-height: 16px;
   letter-spacing: 0.03em;
-  color: #6070ff;
+  color: #1f35ff;
```
Comment (Joaquin123455) [inline style.css:305]: contrast color changed.

File: style.css [hunk 10]
```diff
@@ -317,7 +317,7 @@ span {
...
```
Comment (Joaquin123455) [inline style.css:320]: contrast color changed.

File: style.css [hunk 11]
```diff
@@ -327,13 +327,14 @@ button {
   justify-content: center;
   margin: 16px 0 26px 0;
   padding: 12px 12px;
-  color: #6070ff;
+  color: #1f35ff;
```
Comment (Joaquin123455) [inline style.css:330]: contrast color changed.

File: style.css [hunk 12]
```diff
@@ -327,13 +327,14 @@ button {
...
   box-sizing: border-box;
-  border: 1px solid #6070ff;
+  border: 1px solid #1f35ff;
```
Comment (Joaquin123455) [inline style.css:332]: contrast color changed.

File: style.css [hunk 13]
```diff
@@ -610,7 +611,7 @@ textarea {
   font-weight: 400;
   font-size: 15px;
   line-height: 24px;
-  color: #b3bac5;
+  color: black;
```
Comment (Joaquin123455) [inline style.css:614]: contrast color changed.

File: style.css [hunk 14]
```diff
@@ -626,7 +627,7 @@ textarea {
   font-weight: 500;
   font-size: 17px;
   line-height: 24px;
-  color: #6070ff;
+  color: #1f35ff;
```
Comment (Joaquin123455) [inline style.css:630]: contrast color changed.

**Review by AdnanOlarmmi — APPROVED** [review]
Hi @Joaquin123455,

This is Adnan Is-haq, your beloved code reviewer, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Approved office man

## Highlights
- App is now accessible.
- 😊 Yay! You fixed the changes

## Optional suggestions
_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**